docs/concepts/actions.md

```diff
@@ -77,3 +77,55 @@ Examples:
 
 A great video on the topic is [Good Action Hygiene by Mike Ryan](https://www.youtube.com/watch?v=JmnsEvoy-gY)
 It's for NgRx, but the same naming conventions apply to NGXS.
+
+## Group your actions
+Currently we suffix 'Action' with each action class name like:
+
+```TS
+export class AddAction {
+  static readonly type = '[Todo] Add';
+  constructor(public payload: any) { }
+}
+
+export class EditAction {
+  static readonly type = '[Todo] Edit';
+  constructor(public payload: any) { }
+}
+
+export class FetchAllAction {
+  static readonly type = '[Todo] Fetch All'
+}
+
+export class DeleteAction {
+  static readonly type = '[Todo] Delete';
+  constructor(public id: number) { }
+}
+
+```
+
+here we just add 'Action' with namespace, so we can group similar action.
+In this case just import namespace instead of multiple action class in same file.
+
+```TS
+export namespace TodoAction {
+
+  export class Add {
+    static readonly type = '[Todo] Add';
+    constructor(public payload: any) { }
+  }
+  
+  export class Edit {
+    static readonly type = '[Todo] Edit';
+    constructor(public payload: any) { }
+  }
+  
+  export class FetchAll {
+    static readonly type = '[Todo] Fetch All'
+  }
+  
+  export class Delete {
+    static readonly type = '[Todo] Delete';
+    constructor(public id: number) { }
+  }
+}
+```
```

Comment:
How do these changes relate to this PR?

Reply:
@markwhitfeld I did this one year ago and already merged,  but don't know why this appears.

Reply:
Oh, the process would be that in your repo, you should get the latest from master, then create a branch for the PR and then push changes to this branch. I think it is doing this because you used the same branch for the other PR.
Strange that it is giving conflicts though.